.github/CODEOWNERS

```diff
@@ -39,7 +39,6 @@
 /jest.preset.js @microsoft/fluentui-react-build
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30515&commitId=d8b8134bcecc46f5a40b8803b07b09c1be102cca&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>